truffle-config.js

```diff
@@ -82,15 +84,20 @@ module.exports = {
       provider: () => {
         return new HDWalletProvider(adminKey, bNetworkNode);
       },
-      network_id: "*",
+      network_id: "2020",
     },
-    // Alastria red T connection with provider
+    // Alastria red B connection with provider
     'red-b-first-id': {
       gasPrice: 0x0,
       provider: () => {
         return new HDWalletProvider(firstIdKey, bNetworkNode);
       },
-      network_id: "*",
+      network_id: "2020",
+    },
+    "alastria-id-b": {
+      provider: () => new HDWalletProvider(mnemonic, `http://63.33.206.111:8545`),
```

Comment:
Can we move the IP to something like RED_B_NODE_IP? I mean, could be interesting at some point move the IP configs to a config file or env, its a changing config that does not depends on the network configuration (each member of Alastria will be using their nodes, with the same chain config)